src/azure_queue_service.js

```diff
@@ -62,8 +62,8 @@ const processQueueMessages = async () => {
 
         const outputDir = "/maps";
 
-        // Pass the extracted values to initiateRendering
-        await initiateRendering(
+        // Pass the extracted values to initiateRendering, and receive the metadata
+        const metadata = await initiateRendering(
```

Comment:
nitpick: choose a more specific word. "metadata" can be used for all sorts of stuff.

idea: taskResult, renderResult